src/actions/dateNavActions.js

```diff
@@ -0,0 +1,7 @@
+export const PERIOD_CHANGED = 'PERIOD_CHANGED';
```

Comment:
this action is fired when the user wants to change the day the to fetch entries fore